pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -235,8 +247,21 @@ def cmd(self):
             cmd.append("--verbose")
         if self.disable_capture:
             cmd.append("-s")
-        if self.eval_attr:
-            cmd.append("-a '{}'".format(self.eval_attr))
+
+        if self.xdist_ip_addresses:
+            cmd.append('-d')
```

Comment:
Same comment as above regarding the `--dist` option in use.  I expect some test failures initially here, since we haven't been running these tests in parallel; we'll have to fix any test order dependencies and/or parallelism bugs before moving to this as the default execution mode.